templates/edit.php

```diff
@@ -88,17 +88,24 @@
       </select>
     </td>
   </tr>
+  <?php if(isset($user_data['wprdesknumber'][0]) && $user_data['wprdesknumber'][0] != 'Off-Site') {?>
```

Comment:
Does the database have both a null and an "Off-Site" attribute? They'd mean the same thing, right? 🤔